### DISCUSSION TITLE
Introduce classes for dynamic viewport height/width (dvh/w, lvh/w, svh/w)

### DISCUSSION THREAD

**Discussion by casperjdev** [tailwindlabs/tailwindcss#13814] [Ideas]
`h-screen` aswell as `w-screen` use `vh/w` as their units.

This is not ideal, especially when trying to make a design responsible on mobile browsers.

It would be handy to have classes like:
`w-screen-d`
`h-screen-l`
etc.

Naming might be a bit tricky, but it would be nice to not have to use arbitrary values here.

Alternatively, it could be easier to make `screen` use `dvh/w` and not `vh/w`.

**Comment by adamwathan** [comment]
Hey! We added these in v3.4 in December 👍🏻

https://tailwindcss.com/blog/tailwindcss-v3-4#dynamic-viewport-units

**Reply by casperjdev** [reply]
Hey! Thanks for the swift reply.


I might have missed that while reading the documentation. Thanks!
